lib/core/src/display/shape/text/content.rs

```diff
@@ -124,10 +124,19 @@ impl TextChange {
     }
 }
 
+
+
 // ============================
 // === TextComponentContent ===
 // ============================
 
+/// A position of character in multiline text.
+#[derive(Copy,Clone,Debug,PartialEq,Eq,PartialOrd,Ord)]
+pub struct CharPosition {
```

Comment:
I think that a name `TextLocation` reflects better the intention of this struct.